internal/resources/data_source_morpheus_token_broker.go

```diff
@@ -0,0 +1,60 @@
+// (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+
+package resources
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/client"
+)
+
+// MorpheusDetailsBroker returns a schema.Resource for the MorpheusDetails data source
+func MorpheusDetailsBroker() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"access_token": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Morpheus access_token",
+				Sensitive:   true,
+			},
+			"refresh_token": {
```

Comment:
whats the use of `refresh_token` and how will it be used to make call to morpheus tf

Reply:
It is there in case the Morpheus provider actually supports refresh_token.  I can remove for now.